src/test/rustdoc/implementor-stable-version.rs

```diff
@@ -0,0 +1,18 @@
+#![crate_name = "foo"]
+
+#![feature(staged_api)]
+
+// @has foo/trait.Foo.html '//*[@id="impl-Foo-1"]//span[@class="since"]' '2.0'
+// @!has - '//*[@id="impl-Foo"]//span[@class="since"]' '1.0'
```

Comment:
I'm not super happy about depending on the order the IDs are generated. Could you do this by the name of the struct instead, maybe?

Reply:
How could I do that? This is my very first rustdoc test 😄

Reply:
Hmm, I'm not actually sure, since the `code` block also has other nested markup. @GuillaumeGomez do you have ideas?

Reply:
Oh, I could just have two traits instead

Reply:
I'd try to go through classes if you don't want to go through IDs. In this case, it seems safer not to use IDs. Try to give a path as unique as possible based on the classes and some other attributes.

Reply:
And yes, the simpler the rust code, the simpler the HTML checks. :p